spec/ace.md

```diff
@@ -132,41 +132,62 @@ An executor MAY implement a "strict mode" where an image cannot run unless all i
 These isolators are specific to the Linux kernel and are impossible to represent as a 1-to-1 mapping on other kernels.
 The first example is "capabilities" but this will be expanded to include things such as SELinux, SMACK or AppArmor.
 
-#### os/linux/capabilities-remove-set
+#### os/linux/capabilities-retain-set
 
 * Scope: app
 
 **Parameters:**
 
-* **set** list of capabilities that will be removed from the process's capabilities bounding set, all others will be included.
+* **set** list of capabilities that will be added in the process's capabilities bounding set in addition to the default app capabilities defined thereafter.
```

Comment:
While it is possible for earlier versions of application images may be looking for additional capabilities, my understanding of the retain set from the specification was that it allowed the developer to create a whitelist of capabilities their application should have access to.

Additionally, once the remove set is implemented in rkt it will be possible to have a smaller set of capabilities but it is far easier in cases where a small number of capabilities are required to say:
I need CAP_KILL, and CAP_CHOWN only, rather than specifying the 12 or so capabilities that they do not need from the default capability list.